src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -424,6 +424,27 @@ def subvolume_getpath(self, fs_handle, **kwargs):
             ret = self.volume_exception_to_retval(ve)
         return ret
 
+    @connection_pool_wrap
+    def list_subvolumes(self, fs_handle, **kwargs):
+        ret        = 0, "", ""
+        groupname  = kwargs['group_name']
+
+        try:
+            with SubVolume(self.mgr, fs_handle) as sv:
+                spec = SubvolumeSpec(None, groupname)
+                if not self.group_exists(sv, spec):
+                    raise VolumeException(
+                        -errno.ENOENT, "Subvolume group '{0}' not found".format(groupname))
+                path = sv.get_group_path(spec)
```

Comment:
When is the `path` ever `None`? Why do you need the below check?

Reply:
Don't create a subvolume and try to list subvolumes, `get_group_path` returns None and `sv.get_dir_names()` crashes.

```
[jcollin@stratocaster build]$ ./bin/ceph fs subvolume ls myfs
Error ENOENT: Subvolume group path '/volumes/_nogroup' not found
```